src/persist-client/src/internal/compact.rs

```diff
@@ -1298,103 +964,4 @@ mod tests {
         assert_eq!(part.desc, res.output.desc);
         assert_eq!(updates, all_ok(&data, CodecProduct::new(10, 0)));
     }
-
-    #[mz_ore::test(tokio::test)]
-    #[cfg_attr(miri, ignore)] // unsupported operation: returning ready events from epoll_wait is not yet implemented
-    async fn prefetches() {
```

Comment:
We have a similar test for the streaming consolidator already, so I don't feel bad about deleting this one.